store/transient/store.go

```diff
@@ -14,16 +14,23 @@ var _ types.KVStore = (*Store)(nil)
 // Store is a wrapper for a MemDB with Commiter implementation
 type Store struct {
 	dbadapter.Store
+	version int64
 }
 
 // Constructs new MemDB adapter
 func NewStore() *Store {
 	return &Store{Store: dbadapter.Store{DB: dbm.NewMemDB()}}
 }
 
+// Implements Committer
+func (st *Store) Version() int64 {
+	return st.version
+}
+
 // Implements CommitStore
 // Commit cleans up Store.
 func (ts *Store) Commit() (id types.CommitID) {
+	ts.version += 1
```

Comment:
This always returns an empty id every time?
Is this still included in the root CommitInfo?